mycroft/skills/skill_manager.py

```diff
@@ -303,23 +361,37 @@ def run(self):
 
         self.status.set_alive()
 
-        if self.skills_config.get("wait_for_internet", True):
-            while not connected() and not self._connected_event.is_set():
-                sleep(1)
-            self._connected_event.set()
-
         self._load_on_startup()
 
-        # Sync backend and skills.
-        # why does selene need to know about skills without settings?
-        if is_paired():
-            self.manifest_uploader.post_manifest()
+        if self.skills_config.get("wait_for_internet", False):
+            LOG.warning("`wait_for_internet` is a deprecated option, update to "
+                        "specify `network_skills` and `internet_skills` in "
+                        "`ready_settings`")
+            # NOTE - self._connected_event will never be set
+            # if PHAL plugin is not running to emit the connected events
```

Comment:
we could make the plugin answer to some bus message to confirm its loaded, and if not loaded do something here instead of silently waiting forever?

Reply:
We would have to poll it since we don't know which order service start in.. I just marked this as deprecated since we have ready_settings now to deal with it